### PR TITLE
feat: init notion service

### PR DESCRIPTION
**Pull request by mkhoi1998**
#### What's this PR do?

- [x] init notion service

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -77,6 +82,10 @@ func generateConfigFromViper(v *viper.Viper) *Config {
 			Token:   v.GetString("VAULT_TOKEN"),
 			Path:    v.GetString("VAULT_PATH"),
 		},
+
+		Notion: Notion{
+			Secret: v.GetString("NOTION_SECRET"),
```
Comment (nguyennh4522) [inline pkg/config/config.go:87]: add the same value into `service/vault/vault.go`

File: pkg/service/notion/notion.go [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+		records = append(records, res.Results...)
+		if !res.HasMore {
+			break
+		}
+		cursor = *res.NextCursor
+	}
+	return records, nil
+}
+
+func (n *notionService) QueryAudienceDatabaseByPastWeek(audienceId string) (records []nt.Page, err error) {
```
Comment (huynguyenh) [inline pkg/service/notion/notion.go:291]: tạm thời delete

File: pkg/service/notion/notion.go [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+func (n *notionService) QueryDatabase(id string) (records []nt.Page, err error) {
+	ctx := context.Background()
+	res, err := n.notionClient.QueryDatabase(ctx, id, nil)
+	if err != nil {
+		return nil, err
+	}
+	return res.Results, nil
+}
+
+func (n *notionService) QueryAudienceDatabase(audienceDBId, audience string) (records []nt.Page, err error) {
```
Comment (huynguyenh) [inline pkg/service/notion/notion.go:216]: tạm thời delete

File: pkg/service/notion/notion.go [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+		})
+		if err != nil {
+			return err
+		}
+		number++
+	}
+	return nil
+}
+
+func (n *notionService) Unsubscribe(id, pid string) error {
```
Comment (huynguyenh) [inline pkg/service/notion/notion.go:194]: tạm thời delete

File: pkg/service/notion/notion.go [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+						},
+					},
+				},
+			},
+		},
+	})
+	return p.ID, err
+}
+
+func (n *notionService) CreateOnLeave(id, name, summaryID, tag string, dates []time.Time) error {
```
Comment (huynguyenh) [inline pkg/service/notion/notion.go:114]: tạm thời delete

**Review by huynguyenh — APPROVED** [review]

**Pull request merged**